wayland-commons/src/debug.rs

```diff
@@ -0,0 +1,71 @@
+//! Debugging helpers to handle `WAYLAND_DEBUG` env variable.
+
+use std::time::{SystemTime, UNIX_EPOCH};
+
+use crate::wire::Argument;
+
+/// Print the dispatched event to stderr in a following format:
+///
+/// [timestamp] <- interface@id.msg_name(args)
+pub fn print_dispatched_event(interface: &str, id: u32, msg_name: &str, args: &[Argument]) {
```

Comment:
If you have more clear name, please suggest.

Reply:
`print_dispatched_message` would be more appropriate, as the term covers both requests and events